app/Http/Controllers/Admin/FieldController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Admin\Fields\CreateFieldRequest;
+use App\Http\Requests\Admin\Fields\DeleteFieldRequest;
+use App\Http\Requests\Admin\Fields\EditFieldRequest;
+use App\Models\Field;
+use Illuminate\Http\Request;
+
+class FieldController extends Controller {
+
+    public function index(){
+
+    }
+
+    public function create(CreateFieldRequest $request) {
+        $field = $request->commit();
+        return $field;
+    }
+
+    public function edit(Field $field, EditFieldRequest $request) {
+        $field = $request->edit();
```

Comment:
What is the naming convention we are using? we can either use the name of the action, but then the previous method (in the create method) should be called create - or we change both the edit and delete methods to "commit"

Reply:
i dont care, if you dont either ill just change the create